.github/actionlint-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "actionlint",
+      "pattern": [
+        {
+          "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
```

Comment:
[q] Why is this file needed? Could you add a comment in the file to provide a matching string?

Reply:
>Could you add a comment in the file to provide a matching string?

json doesn't support comments :) 

This files enables the GH annotations and used in the GH action like
```
echo "::add-matcher::.github/actionlint-matcher.json"
```
https://github.com/rhysd/actionlint/blob/main/docs/usage.md#problem-matchers

Reply:
oh nice thanks!